app/Models/Post.php

```diff
@@ -233,4 +233,29 @@ public function getTotalReactions($postId)
     {
         return Reaction::where('post_id', $postId)->count();
     }
+
+    /**
+     * Get the total quantity for all posts under the signup that this post is associated with.
+     *
+     * @return int
+     */
+    public function getTotalQuantity()
+    {
+        $signup = $this->signup;
+
+        // If the post has NULL quantity, it means that the quantity is still on the signup
+        if (is_null($this->quantity)) {
```

Comment:
oh wait - one question @katiecrane ! What happens if there is `quantity_pending` on the signup?

Reply:
also, do we want to return the `quantity` if it isn't `null`?

Reply:
ahh shoot! I gotta add a check for that!